service/src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -627,6 +631,77 @@ public boolean resourceExists(UUID workspaceUuid, UUID resourceId) {
     return (count != null && count > 0);
   }
 
+  @WriteTransaction
+  public boolean deleteResourceProperties(
```

Comment:
`deleteResourceProperties()` and `updateResourceProperties()` are not used. They should be removed from this PR.

Reply:
i added this because there's no way to test properties given they are always empty lol but that's not a good reason. i will merge it with my folder change.

Reply:
To test, can't you set the properties field when you create a resource? And this PR has the logic for copying resource properties when you clone. (At least you did this for controlled, did you do this for referenced too?) That's why I suggested testing cloning a workspace or resource. That makes sure that 1) Property correctly set on source resource, 2) property copied over to dest resource.

Reply:
that's the thing.. i intentionally left it out of the creation path because that's what workspace properties does. It will have separate endpoint for properties update/delete. 

i added aunit test in workspacecloneutils.

yeah. it's automatically done for referenced resource through toBuilder.

Reply:
Ah... now I understand why you had the dao methods. Sorry. No tests are fine then.

Now that I understand there's no other way to test, the dao methods were fine. (Fine to leave them out too.)